src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -238,7 +238,7 @@ const SimOptionsPage = () => {
     const [fpSync, setFpSync] = usePersistentProperty('FP_SYNC', 'LOAD');
     const [dynamicRegistration, setDynamicRegistration] = usePersistentProperty('DYNAMIC_REGISTRATION_DECAL', '0');
     const [defaultBaro, setDefaultBaro] = usePersistentProperty('CONFIG_INIT_BARO_UNIT', 'AUTO');
-    const [mcduServerPort, setMcduServerPort] = usePersistentProperty('CONFIG_EXTERNAL_MCDU_PORT', '8380');
+    const [mcduServerPort, setMcduServerPort] = usePersistentProperty('CONFIG_EXTERNAL_MCDU_PORT', '3838');
```

Comment:
Each time we change this we break our users bookmarks/home-cockpits etc. Is it strictly necessary?

Reply:
This was just a modification from the original mcdu-server, might be useful to ask @tyler58546